projects/storefrontstyles/scss/themes/calydon/components/product/_index.scss

```diff
@@ -0,0 +1 @@
+@import './list/index';
```

Comment:
You need to add a trailing blank line on each file so prettier doesn't complain.